dds/idl/be_global.cpp

```diff
@@ -348,6 +348,9 @@ BE_GlobalData::parse_args(long& i, char** av)
   static const char EXPORT_FLAG[] = "--export=";
   static const size_t EXPORT_FLAG_SIZE = sizeof(EXPORT_FLAG) - 1;
 
+  static const char TYPESEQUENCESUFFIC_FLAG[] = "--type-sequence-suffix=";
```

Comment:
```suggestion
  static const char TYPESEQUENCESUFFIX_FLAG[] = "--type-sequence-suffix=";
```

Change the other uses of this.

Also add `--type-sequence-suffix` to [`--help`](https://github.com/objectcomputing/OpenDDS/blob/28ce33c4ff01963861abce7a3f77c0ef865c8117/dds/idl/be_util.cpp#L99).